index.js

```diff
@@ -548,20 +605,72 @@ class KeyringController extends EventEmitter {
    * @param {string} password - The keyring controller password.
    * @returns {Promise<Array<Keyring>>} The keyrings.
    */
-  async unlockKeyrings(password) {
+  async unlockKeyrings(password, encryptedKey) {
```

Comment:
:thinking: Why is it called encryptedKey not encryptionKey? It's not like the key itself is encrypted.